ext/libmemcached-0.32/libmemcached/memcached_connect.c

```diff
@@ -14,7 +14,7 @@ static memcached_return set_hostinfo(memcached_server_st *server)
 
   memset(&hints, 0, sizeof(hints));
 
- // hints.ai_family= AF_INET;
+  hints.ai_family= AF_INET;
```

Comment:
Seems like this could be turned into an opt-in feature for libmemcached so that it could be proposed upstream.  Doing so would also allow the application code to explicitly opt-in to disabling IPv6, since patching the vendored libmemcached in our fork of the gem makes it not clear in the application configuration that this is disabled.